aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,56 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Overview of the use of SignalR's HubContext
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 6/12/18
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+
+# Sending Messages Outside of you Hub Class
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+The SignalR Hub is the core abstraction for sending messages to clients connectied to our SignalR server. You aren't limited to sending messages to clients from within a Hub though. Here we'll look at how to access a SignalR HubContext to send notification to clients from outside of a hub.
```

Comment:
nit: connectied -> connected

Reply:
You aren't limited to sending messages to clients from within a Hub though. Here we'll look at how to access a SignalR HubContext to send notification to clients from outside of a hub.

->

However, it is also possible to send messages from other places in your application using the `IHubContext` service.